src/main/java/in/projecteka/consentmanager/consent/model/request/ConsentApprovalRequest.java

```diff
@@ -11,4 +11,7 @@
     @Valid
     @NotNull(message = "Consents are not specified")
     private List<GrantedConsent> consents;
+    @Valid
+    @NotNull(message = "Callback Url not specified")
+    private String callBackUrl;
```

Comment:
Why it's part of ConsentApprovalRequest? shouldn't it be part of ConsentRequest from HIU?